src/System.Memory/ref/System.Memory.cs

```diff
@@ -92,15 +92,48 @@ public static partial class MemoryExtensions
         public static int ToLowerInvariant(this System.ReadOnlySpan<char> source, System.Span<char> destination) { throw null; }
         public static int ToUpper(this System.ReadOnlySpan<char> source, System.Span<char> destination, System.Globalization.CultureInfo culture) { throw null; }
         public static int ToUpperInvariant(this System.ReadOnlySpan<char> source, System.Span<char> destination) { throw null; }
+        public static System.Memory<char> Trim(this System.Memory<char> memory) { throw null; }
+        public static System.ReadOnlyMemory<char> Trim(this System.ReadOnlyMemory<char> memory) { throw null; }
         public static System.ReadOnlySpan<char> Trim(this System.ReadOnlySpan<char> span) { throw null; }
+        public static System.Span<char> Trim(this System.Span<char> span) { throw null; }
```

Comment:
I had to _manually_ add the **9** new `TrimXXX<char>` methods, since `dotnet msbuild /t:GenerateReferenceSource` would not pick them up??

Reply:
Ignore, fixed. Local build was stale